internal/store/endpoint.go

```diff
@@ -151,6 +151,34 @@ func endpointMetricFamilies(allowAnnotationsList, allowLabelsList []string) []ge
 				}
 			}),
 		),
+		*generator.NewFamilyGenerator(
+			"kube_endpoint_address",
+			"Information about Endpoint available and non available addresses.",
+			metric.Gauge,
+			"",
+			wrapEndpointFunc(func(e *v1.Endpoints) *metric.Family {
+				ms := []*metric.Metric{}
+				for _, s := range e.Subsets {
+					for _, available := range s.Addresses {
+						ms = append(ms, &metric.Metric{
+							LabelValues: []string{available.IP},
+							LabelKeys:   []string{"available_ip"},
```

Comment:
Should we have a single `ready` label which has `true` or `false` values?

Reply:
> Should we have a single ready label which has true or false values?

yep, make sense.
should we than change `available_ip` and `unavailable_ip` to `ip`?

Reply:
Yes, I think making that change would be consistent with the `ready` label.